packages/block-library/src/blocks/hero/edit.js

```diff
@@ -149,20 +149,22 @@ class HeroEdit extends Component {
 				<InspectorControls>
 					<LayoutControls { ...this.props } />
 					<BlockHeightControls { ...this.props } />
+					<ControlsSection label={ __( 'Display' ) } group={ __( 'Block Modules' ) }>
+						<ControlsTab label={ __( 'Settings' ) }>
+							<ControlsGroup title={ __( 'Set up elements for this block', '__plugin_txtd' ) }>
+								<ToggleGroup
+									onChange={ updateAttributes }
+									toggles={ toggles.map( toggle => {
+										return {
+											...toggle,
+											value: attributes[ toggle.attribute ]
+										}
+									} ) }
+								/>
+							</ControlsGroup>
+						</ControlsTab>
+					</ControlsSection>
```

Comment:
`<ControlsSection>` should stay outside of `<InspectorControls>` as they are both Fills and the **ControlsSections** slot is already inside the **InspectorControls** Fill used in the `with-controls-sections` filter that is applied on all blocks.